src/chapters/01-orientation.md

```diff
@@ -145,19 +161,46 @@ ember server | awk '{ \
 ember server
 ```
 
+The development server is responsible for compiling our app and serving it to the browsers. It may take a while to boot up, but once it's up an running, open your favorite browser and head to http://localhost:4200. You should see the following welcome page:
+
 ```run:screenshot width=1024 retina=true filename=welcome.png alt="Welcome to Ember!"
 visit http://localhost:4200/
 ```
 
-<!-- TODO: explain ember server, etc. -->
+> Zoey says...
+>
+> The `localhost` address in URL means that you can only access the development server from your local machine. If you would like to share your work to the world, you will have to *[deploy][TODO: link to deploy]* your app to the public Internet. Don't worry, we will cover that in Part 2 of the tutorial.
+
+You can exit out of the development server anytime but pressing `Ctrl + C` on your keyboard in the terminal which the server is running in. It is recommended that you run the development server in a new, dedicated terminal window. That way, you can keep it running in the background and won't have to stop and start the server everytime you want to run a command.
```

Comment:
The sentences "It is recommended ... to run a command." might come across as a bit confusing to developers who are not used to Ember's live reload. I think we can more concisely describe the practice of having two tabs (terminals) open, one to run the server, and the other to type commands. 
```suggestion
You can exit out of the development server at any time by pressing `Ctrl + C` on your keyboard in the terminal where the server is running. We recommend having two terminal windows open: one to run the server in background, another to type Ember CLI commands.
```